src/main.js

```diff
@@ -145,57 +151,123 @@ let load = base.loader({
     },
   },
 
-  queue: {
-    requires: ['cachemanagers', 'cfg', 'sqs', 'profile', 'monitor'],
-    setup: async ({cachemanagers, cfg, sqs, profile, sqsQueues, monitor}) => {
+  initQueue: {
+    requires: ['cfg', 'sqs'],
+    setup: async ({cfg, sqs}) => {
+      let sqsConfig = cfg.sqsSimple;
+      sqsConfig.sqs = sqs;
+      debug(sqsConfig);
+      return sqsSimple.initQueue(sqsConfig);
+    },
+  },
+
+  queueUrl: {
+    requires: ['cfg', 'sqs'],
+    setup: async ({cfg, sqs}) => {
+      return sqsSimple.getQueueUrl({
+        sqs,
+        queueName: cfg.sqsSimple.queueName,
+      });
+    },
+  },
 
-      let handler = async (obj) => {
-        assert(obj.id, 'must provide id in queue request');
-        assert(typeof obj.id === 'string', 'id must be string');
-        assert(obj.url, 'must provide url in queue request');
-        assert(typeof obj.url === 'string', 'url must be string');
+  deadQueueUrl: {
+    requires: ['cfg', 'sqs', 'profile'],
+    setup: async ({cfg, sqs, profile}) => {
+      return sqsSimple.getQueueUrl({
+        sqs,
+        queueName: cfg.sqsSimple.queueName + cfg.sqsSimple.deadLetterSuffix,
+      });
+    },
+  },
 
-        let selectedCacheManagers = cachemanagers.filter(x => x.id === obj.id);
-        await Promise.all(selectedCacheManagers.map(x => x.put(obj.url)));
-      };
+  queueSender: {
+    requires: ['cfg', 'sqs', 'queueUrl'],
+    setup: async ({cfg, sqs, queueUrl}) => {
+      return new sqsSimple.QueueSender({sqs, queueUrl});
+    },
+  },
+
+  queueListenerFactory: {
```

Comment:
i was thinking that, but i don't really know where to put it.  While it's this simple, let's leave it in main.js and revisit the next time we change the handler.  I'd like to move deleting from the front end into backend, so that'll be a good time for this change.